### PR TITLE
suppress sending telemetry of error details

### PR DESCRIPTION
**Pull request by hellyzh**
1.update dependency "vscode-extension-telemetry" to version 0.1.6
2. set firstParty = true for TelemetryReporter.
3. use sendTelemetryErrorEvent to suppress sending telemetry of error details.
4. update name of telemetry with error from "error" to predefined message.

### CODE REVIEW

File: src/logger/telemetry-transport.ts [hunk 1]
```diff
@@ -60,7 +60,11 @@ export class TelemetryTransport extends winston.Transport {
                         }
                     }
                 }
-                this.reporter.sendTelemetryEvent(message, properties, measures);
+                if (level === "info") {
```
Comment (dilin-MS) [inline src/logger/telemetry-transport.ts:63]: Could you please set "info" "error" and "warn" as an enum like "LogLevel" and replace the hard-coded string in the files?

Comment (hellyzh) [reply src/logger/telemetry-transport.ts]: updated.

File: src/logger/logger.ts [hunk 2]
```diff
@@ -6,6 +6,12 @@ import * as winston from "winston";
 import TelemetryTransport from "./telemetry-transport";
 import UserNotificationTransport from "./user-notification-transport";
 
+export enum LogLevel {
+    info = "info",
```
Comment (dilin-MS) [inline src/logger/logger.ts:10]: Please follow the typescript [coding guidelines](https://github.com/microsoft/TypeScript/wiki/Coding-guidelines#names) and use PascalCase for enum values.

Comment (hellyzh) [reply src/logger/logger.ts]: Updated enum values to PascalCase

**Pull request merged**